index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/indexStyles.css">
+    <link rel="stylesheet" href="./styles/blogStyles.css">
+    <title>Document</title>
+</head>
+<body class="body__style">
+    <div class="grand_container__style">
+        <h1 class="h1_grand_container__style">Willow Crest</h1>
```

Comment:
and here as well. Widely spread mistakes. Please, re-check your code and rename all bad-named classes. Thanks